### DISCUSSION TITLE
Lost Canva Pro

### DISCUSSION THREAD

**Discussion by Alessa055** [community/community#45115] [GitHub Education]
### Select Topic Area

Question

### Body

Hi Everyone!
What can I do if I lost my Canva pro subscription? It´s still connected to Github but it´s no upgradet to pro anymore
In my account it says I don´t need to verify that I´m a student, but I don´t know what to do in this case
Can anyone help me?
Sorry for the english and grammar, it´s not my first language.
Thank you!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @Alessa055 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
